### PR TITLE
Feature/115972461 nk upgrade monitoring to v3

### PR DESCRIPTION
**Pull request by krakover**
https://www.pivotaltracker.com/story/show/115972461

### CODE REVIEW

**Comment by liorr** [comment]
The builds still fails

**Comment by krakover** [comment]
It passes locally. maybe the pip/setuptools versions on the machines' are not updated?

**Comment by liorr** [comment]
If it passes locally I am sure that the problem is locally on your machine

File: googleservices/cloudmonitoring/client.py [hunk 1]
```diff
@@ -28,29 +28,33 @@ def write_timeseries_custon_instance_metric_double_value(self, project_id, metri
...
+            'timeSeries': [{
+                'metric': {
+                    'type': 'custom.googleapis.com/custom/%s' % metric_name,
+                    'labels': {},
+                },
+                'resource': {
+                    'type': 'gce_instance',
+                    'labels': {
+                        'instance_id': unique_machine_id,
+                        'zone': 'us-central1-a'}
```
Comment (liorr) [inline googleservices/cloudmonitoring/client.py]: I would probably add the zone as a parameter to this function, we might have machines from different zones

File: googleservices/cloudmonitoring/client.py [hunk 2]
```diff
@@ -28,29 +28,33 @@ def write_timeseries_custon_instance_metric_double_value(self, project_id, metri
...
+                },
+                'metricKind': 'GAUGE',
+                'valueType': 'double',
```
Comment (liorr) [inline googleservices/cloudmonitoring/client.py]: wouldn't INT64 be better here?

File: googleservices/cloudmonitoring/client.py [hunk 3]
```diff
@@ -28,29 +28,33 @@ def write_timeseries_custon_instance_metric_double_value(self, project_id, metri
         unique_machine_id = get_gce_unique_id(http)
```
Comment (liorr) [inline googleservices/cloudmonitoring/client.py:28]: What is this comment here?

Comment (krakover) [inline googleservices/cloudmonitoring/client.py:28]: Here I am starting the build the post data for this http request. The nicer way will be to you the client API (that I tried to update for this purpose), but it doesn't support it yet (the client api wraps the request so I will not have to). so I have to do it myself until they release a new version that has it.

**Pull request merged**